tests/cypress/integration/newfold-labs/wp-module-onboarding/steps/getstarted-welcome.cy.js

```diff
@@ -0,0 +1,36 @@
+// <reference types="Cypress" />
```

Comment:
Can we add a few more tests
1. We are testing clicks for the tabs, but in a normal browser user would hover over them to change the tab content. Can we test `hover` as well?
2. We are testing if the drawer is not visible, we should also be testing if clicking on the `drawer-toggle` button does not open the drawer.
3. Test if the Navigation Next button goes to the next step.
4. Since we have access to wp-cli we can set the `mm_brand` to `BlueHost` and see if `Bluehost` shows up in the subheading.

Reply:
Pushed the changes
1. Cypress does not have a hover api so have user trigger
2. drawer toggle test added
3. next step test added
4. brand name check added
5. exit to wordpress test added, have placed the exit test last since after that go.('back') does not return back the user to the welcome step
6. added the cy.setCustomerData() and cy.clearCustomerData() methods